.env.development

```diff
@@ -10,7 +10,7 @@ DB_NAME='bhima_test'
 # session variables
 SESS_SECRET='XopEn BlowFISH'
 
-DEBUG=db
+DEBUG=app
```

Comment:
What is the meaning of this? can you comment?

Reply:
This changes the log level to only show when the app has started and loaded environmental variables.  Previously, @mbayopanda set it to print every db execution making the environmental variable "db".  The problem with the "db" value is that it overflows Travis's logs and makes them unreadable.  So I was switching it back to its default behavior.

The way debug works is described in its README.  You can read all about it [here](https://www.npmjs.com/package/debug).  It's worth reading for when we have to debug errors in production!